runtime/src/accounts_db.rs

```diff
@@ -54,18 +54,18 @@ pub const DEFAULT_NUM_DIRS: u32 = 4;
 
 #[derive(Debug, Default)]
 pub struct ErrorCounters {
-    pub account_not_found: usize,
+    pub total: usize,
     pub account_in_use: usize,
     pub account_loaded_twice: usize,
+    pub account_not_found: usize,
     pub blockhash_not_found: usize,
     pub blockhash_too_old: usize,
-    pub reserve_blockhash: usize,
```

Comment:
hehe was `blockhash_not_found` just not being used before? yikes

Reply:
ha yea